File: Warmup/Plus Minus/Plus Minus.cs
using System.CodeDom.Compiler;
using System.Collections.Generic;
using System.Collections;
using System.ComponentModel;
using System.Diagnostics.CodeAnalysis;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Reflection;
using System.Runtime.Serialization;
using System.Text.RegularExpressions;
using System.Text;
using System;

class Solution {
   /*
     * Complete the 'plusMinus' function below.
     *
     * The function accepts INTEGER_ARRAY arr as parameter.
     */

    static void plusMinus(int arr_count, int[] arr) {
        float pos=0,neg=0,zer=0;
        for(int i=0;i<arr_count;i++){
            if(arr[i]>0){
                pos++;
            }
            else if(arr[i]<0){
                neg++;
            }
            else{
                zer++;
            }
        }
        Console.WriteLine("{0:F6}\n{1:F6}\n{2:F6}",pos/arr_count,neg/arr_count,zer/arr_count);
    }
    static void Main(string[] args) {
        int n = Convert.ToInt32(Console.ReadLine());
        int[] arr = Array.ConvertAll(Console.ReadLine().Split(' '), arrTemp => Convert.ToInt32(arrTemp));
        plusMinus(n, arr);
    }
}

File: Warmup/Plus Minus/Plus Minus.java
import java.io.*;
import java.math.*;
import java.security.*;
import java.text.*;
import java.util.*;
import java.util.concurrent.*;
import java.util.function.*;
import java.util.regex.*;
import java.util.stream.*;
import static java.util.stream.Collectors.joining;
import static java.util.stream.Collectors.toList;

public class Solution {

    /*
     * Complete the 'plusMinus' function below.
     *
     * The function accepts INTEGER_ARRAY arr as parameter.
     */

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        float pos = 0, neg = 0, zer = 0;
        int[] z = new int[n];
        for (int i = 0; i < n; i++) {
            z[i] = scanner.nextInt();
            if (z[i] > 0) {
                pos++;
            } else if (z[i] < 0) {
                neg++;
            } else {
                zer++;
            }
        }
        System.out.printf("%5f\n%5f\n%5f\n", pos / n, neg / n, zer / n);
        scanner.close();
    }
}
